### DISCUSSION TITLE
Change default colour on note map

### DISCUSSION THREAD

**Discussion by the-LONS-network** [TriliumNext/Trilium#3443] [Q&A] [answered]
It's not clear anywhere in the wiki if this can be done but from what I've seen on the [map entry](https://github.com/zadam/trilium/wiki/Note-map), notes can show up as different colours.

So if I had a note with an attribute (ie `#example`), how can I make it show up on the note map as purple instead of light blue?

I expect it's some form of CSS which I'm missing maybe? or is it baked into the code for the map?

Also, on the wiki, its not exactly clear on _how_ to do things so maybe a rewrite could be beneficial.

**Comment by zadam** [comment, marked as answer]
Hi, there's a [`#color`](https://github.com/zadam/trilium/wiki/Attributes) label which changes the note's color in the tree and in links, but so far it did not affect the notes. I fixed that now in master and this will be respected there as well.

The existing colors are based on the note's type.

> So if I had a note with an attribute (ie #example), how can I make it show up on the note map as purple instead of light blue?

It's not possible to define "notes with `#example` should be displayed with such and such color", but it's possible via a [template](https://github.com/zadam/trilium/wiki/Template) in combination with the mentioned `color` label. In the next release, I plan to make this user-friendly - being able to "mark" (assign a template) a note with a mouse/menu.
